### ISSUE TITLE
paddleOCR过拟合

### ISSUE THREAD

**Issue by LinXin04** [PaddlePaddle/Paddle#34790]
使用paddle OCR  在日文小语种finetuning 的过程中，发现训练的acc可以到80%，但是eval的acc只有30%-40%

用了数据增强：
![image](https://user-images.githubusercontent.com/25193686/128961871-7d22bc59-b38f-4d9d-8c7f-7b8c75f7a2ab.png)

用了正则化：
![image](https://user-images.githubusercontent.com/25193686/128961904-49fd662b-cfd1-4e3e-bad8-84476e3fa410.png)

增加了数据量：
1、用了ICDAR的5k个日文的数据集
2、用了自己数据2万

都存在过拟合，不清楚问题在哪

**Comment by LinXin04** [comment]
@yaoxuefeng6 麻烦大佬给看看～

**Comment by LinXin04** [comment]
配置文件
Global:
  distort: True
  use_gpu: True
  epoch_num: 250
  log_smooth_window: 20
  print_batch_step: 20
  save_model_dir: ./output/rec_japan_lite
  save_epoch_step: 20
  eval_batch_step: [0, 2000]
  cal_metric_during_train: True
  load_static_weights: False
  pretrained_model: ./pretrain_models/japan_mobile_v2.0_rec_train/best_accuracy
  checkpoints: 
  save_inference_dir:
  use_visualdl: False
  infer_img:
  character_dict_path: ./ppocr/utils/dict/japan_dict.txt
  character_type: japan
  max_text_length: 25
  infer_mode: False
  use_space_char: False


Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0005
  regularizer:
    name: 'L2'
    factor: 0

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: small
    small_stride: [1, 2, 2, 2]
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 48
  Head:
    name: CTCHead
    fc_decay: 0

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./
    label_file_list: ["./train_data/rec/rec_gt_train_format.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - RecAug: 
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 40
    drop_last: True
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./
    label_file_list: ["./train_data/rec/rec_gt_test_format.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 40
    num_workers: 4

**Comment by littletomatodonkey** [comment]
你好，看着训练集和测试集差距比较
1. 排查下训练和测试集分布是否一致
2. 如果分布基本一致的话，可以试着把regularizer的factor和ctc head的fc decay调大一点，可以参考先试下1e-5或者4e-5

**Comment by LinXin04** [comment]
@littletomatodonkey 

你好，我是在下面这个预训练模型上做的fine-tuning。
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.1/doc/doc_ch/multi_languages.md#%E8%AF%AD%E7%A7%8D%E7%BC%A9%E5%86%99

![image](https://user-images.githubusercontent.com/25193686/128981252-925bfbc6-8496-447f-b75c-8636dc0f8793.png)


刚才认真看了一下该模型文件中的train.log，发现他本身的文件里就是训练train的acc在90%+，eval的best metirc只有在38%，是我对这两个输出理解错了吗？

![image](https://user-images.githubusercontent.com/25193686/128981169-0e5f0862-d045-486d-8080-fb23a44e0b0a.png)

**Comment by littletomatodonkey** [comment]
没有理解错，可以按照上面两个步骤调试一下
